README.md

```diff
@@ -20,14 +16,18 @@
 * [0002: Paketo Governance](./text/0002-governance.md)
 * [0004: Leiningen (Clojure) Buildpack](./text/0004-clojure.md)
 * [0005: Ruby Paketo Buildpack Promotion](./text/0005-ruby-promotion.md)
+* [0006: Web Servers Buildpack Subteam](./text/0006-web-servers.md)
```

Comment:
I would argue this isn't done until https://github.com/paketo-community/web-servers/ has a valid `buildpack.toml` that produces a buildpack as outlined in the RFC.